manual/cypher/cypher-docs/src/docs/dev/ql/load-csv/index.asciidoc

```diff
@@ -2,11 +2,41 @@
 = Load CSV
 
 [abstract]
+--
 `LOAD CSV` is used to import data from CSV files.
-Supports resources compressed with `gzip`, `Deflate`, as well as `ZIP` archives.
-
-The URL of the CSV file is specified by using `FROM` followed by an arbitrary expression evaluating to the URL in question.
-It is required to specify an identifier for the CSV data using `AS`.
+--
+
+* The URL of the CSV file is specified by using `FROM` followed by an arbitrary expression evaluating to the URL in question.
+* It is required to specify an identifier for the CSV data using `AS`.
+* `LOAD CSV` supports resources compressed with _gzip_, _Deflate_, as well as _ZIP_ archives.
+* CSV files can be stored on the database server and are then accessible using a `file:///` URL.
+  Alternatively, `LOAD CSV` also supports accessing CSV files via _HTTPS_, _HTTP_, and _FTP_.
+
+[NOTE]
+.Configuration settings for file URLs
+====
+<<config_allow_file_urls>>::
+This setting determines if Cypher will allow the use of `file:///` URLs when loading data using `LOAD CSV`. Such URLs
+identify files on the filesystem of the database server. Default is _true_.
+
+<<config_dbms.security.load_csv_file_url_root>>::
+Sets the root directory for `file:///` URLs used with the Cypher `LOAD CSV` clause.
+This must be set to a single directory on the filesystem of the database server, and will make all requests to load from `file:///` URLs relative to
+the specified directory (similar to how a unix chroot operates). By default, this setting is not configured.
++
+* When not set, file URLs will be resolved as relative to the root of the database server filesystem.
+  If this is the case, a file URL will typically look like `file:///home/username/myfile.csv` or `file:///C:/Users/username/myfile.csv`.
```

Comment:
If the second example is meant to mimic a Windows path, it should use backspaces.